src/Code.js

```diff
@@ -4,6 +4,7 @@ import { withTheme } from 'styled-components'
 import SyntaxHighlighter, { registerLanguage } from "react-syntax-highlighter/prism-light"
 import defaultTheme from 'react-syntax-highlighter/styles/prism/atom-dark'
 import javascript from 'react-syntax-highlighter/languages/prism/javascript'
+import jsx from 'react-syntax-highlighter/languages/prism/jsx'
```

Comment:
Whoops! I did this in my branch as well...

Reply:
All good. I'll close this PR.